hwdb.d/60-sensor.hwdb

```diff
@@ -226,8 +226,9 @@ sensor:modalias:acpi:BOSC0200*:dmi:*:svnChuwi*:pnHi13:*
 # Chuwi HiBook does not have its product name filled, so we
 # match the entire dmi-alias, assuming that the use of a BOSC0200 +
 # bios-version + bios-date combo is unique
-sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.11:bd05/07/2016:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnHampoo:rnCherryTrailCR:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:*
-sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.11:bd05/28/2016:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnHampoo:rnCherryTrailCR:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:*
+# '*' in ":*svn" is there because newer kernels have inserted a br field there
```

Comment:
well, "newer"? can you be mor specific maybe? moreover, as i understand it very new kernels moved it to the back now, so it's gone again from the middle.

Reply:
Newer are kernels >= 5.8. I've just pushed a new version replacing the "newer" in the comment with ">= 5.8" to make this explicit.

And no the br + efr fields introduced in 5.8 were not moved to the back, because we have some matches which depend on them being where they are (which also gets replaced with a '*' here) in shipping systemd releases, so moving those 2 fields which are used "out there in the wild" would lead to regressions.

The sku field which is a third new field was recently added was moved to the back. The sku field is mentioned in the updated commit message, but otherwise this commit assumes that the sku field will always be at the end (so basically ignores the sku field problem) since that was quickly resolved on the kernel side and has been fixed in the stable series of all affected kernels (5.13.y and 5.14.y series).